### DISCUSSION TITLE
WAF Detection with nuclei scan

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/nuclei#2334] [Q&A]
## Problem

While running nuclei scan with public / private template, many times some payload (`xss`/`lfi`/`rce`) triggers WAF on the target system which results into:
- Vulnerability is not being acutally checked
- User is not informed / aware of it.

## Solution?

As a user, I'm interested in **knowing** which `host:template` scan resulted into WAF trigger for the further review / manual testing / template improvements.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
